src/Email/Service/MailgunSender.php

```diff
@@ -24,44 +25,51 @@ function ($email) {
     }
 
     /**
-     * @param array $to
+     * @param array  $to
      * @param string $fromEmail
      * @param string $fromName
      * @param string $subject
      * @param string $template
-     * @param array $parameters
-     * @param array $cc
-     * @param array $bcc
+     * @param array  $parameters
+     * @param array  $cc
+     * @param array  $bcc
+     * @param bool   $inlineCss
      * @return mixed
+     * @throws \Exception
      */
-    function sendTemplate(array $to, $fromEmail, $fromName, $subject, $template, $parameters = [], $cc = [], $bcc = [])
+    function sendTemplate(array $to, $fromEmail, $fromName, $subject, $template, $parameters = [], $cc = [], $bcc = [], $inlineCss = true)
```

Comment:
You need to add this to the interface as well (and the other email services, even if they don't use it yet)